pom.xml

```diff
@@ -334,12 +334,12 @@
         <version.org.wildfly.bom-builder-plugin>2.0.8.Final</version.org.wildfly.bom-builder-plugin>
         <version.org.wildfly.checkstyle-config>1.0.8.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.common>1.6.0.Final</version.org.wildfly.common>
-        <version.org.wildfly.galleon-plugins>7.2.1.Final</version.org.wildfly.galleon-plugins>
+        <version.org.wildfly.galleon-plugins>7.3.1.Final</version.org.wildfly.galleon-plugins>
```

Comment:
I was told by @fabiobrz and @jbliznak that we are waiting for this line to be merged to ublock our testing with wildfly snapshot.

Please can we work to merge this and unblock wildfly snapshot testing.